README.md

```diff
@@ -95,6 +96,8 @@ optional arguments:
 
 Finds and deduplicate files.
 
+**Important Warning:** This tool only compares timestamps and not file content. Updating a duplicate file in a mergerfs pool will change the content of a single file but the timestamp of all copies. Running dedupping in such conditions may cause dataloss.
```

Comment:
This is not accurate. It does not compare timestamps. It doesn't compare anything unless requested. It looks up all files for a single relpath and then will act according to the dedup method requested. Optionally ignoring batches based on the ignore option.

Reply:
regarding the touch point, i tested by doing touch and by simply editing a file in the pool that was duplicated, both test cases updated modified timestamp on both files so it seemed that this was the behavior. 

So per your description it should be:

**Important Warning:** This tool will only compare the property specified in the deduping method. Updating a duplicate file in a mergerfs pool will change the content of a single file but the timestamp of all copies. Running deduping with newest method under those conditions may cause data loss has the file removed will be "random".

I don't know enough about github to know if i can edit a pull request though. But I'll try to find out.

Reply:
"simply editing a file" How? If you `echo foo > file` it will **not** update the other files. If you don't want all files modified when other actions are taken you should change the policy for the action.